apps/backend/src/lambdas/reconcile.ts

```diff
@@ -32,7 +37,7 @@ export const handler = async (event: SNSEvent, _context?: Context) => {
   const notificationService = app.get(NotificationService);
   const snsService = app.get(SnsManagerService);
   const reportingService = app.get(ReportingService);
-  const numDaysToReconcile = 31;
+  const numDaysToReconcile = 14;
```

Comment:
I understand this difference probably isn't meaningful in the grand scheme, but can I ask **why** this is a needed change?

Reply:
(I do think we should decrease the no. of days we're reconciling for but will need the team/Aiden to agree :) )